android/src/main/java/com/tailscale/ipn/ui/view/AboutView.kt

```diff
@@ -0,0 +1,115 @@
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalUriHandler
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import com.tailscale.ipn.BuildConfig
+import com.tailscale.ipn.R
+
+@Composable
+fun AboutView() {
+    Surface(color = MaterialTheme.colorScheme.surface) {
+        Column(
+            verticalArrangement = Arrangement.spacedBy(
+                space = 20.dp, alignment = Alignment.CenterVertically
+            ),
+            horizontalAlignment = Alignment.CenterHorizontally,
+            modifier = Modifier
+                .fillMaxWidth()
+                .fillMaxHeight()
+                .safeContentPadding()
+        ) {
+            Image(
+                modifier = Modifier
+                    .width(100.dp)
+                    .height(100.dp)
+                    .clip(RoundedCornerShape(50))
+                    .background(Color.Black)
+                    .padding(15.dp),
+                painter = painterResource(id = R.drawable.ic_tile),
+                contentDescription = stringResource(R.string.app_icon_content_description)
+            )
+            Column(
+                verticalArrangement = Arrangement.spacedBy(
+                    space = 2.dp, alignment = Alignment.CenterVertically
+                ), horizontalAlignment = Alignment.CenterHorizontally
+            ) {
+                Text(
+                    stringResource(R.string.about_view_title),
+                    fontWeight = FontWeight.SemiBold,
+                    fontSize = MaterialTheme.typography.titleLarge.fontSize,
+                    color = MaterialTheme.colorScheme.primary
+                )
+                Text(
+                    text = BuildConfig.VERSION_NAME,
+                    fontWeight = MaterialTheme.typography.bodyMedium.fontWeight,
+                    fontSize = MaterialTheme.typography.bodyMedium.fontSize,
+                    color = MaterialTheme.colorScheme.secondary
+                )
+            }
+            Column(
+                verticalArrangement = Arrangement.spacedBy(
+                    space = 4.dp, alignment = Alignment.CenterVertically
+                ), horizontalAlignment = Alignment.CenterHorizontally
+            ) {
+                OpenURLButton(
+                    stringResource(R.string.acknowledgements),"https://tailscale.com/licenses/apple"
+                )
+                OpenURLButton(
+                    stringResource(R.string.privacy_policy), "https://tailscale.com/privacy-policy/"
```

Comment:
Sometimes I put these kinds of urls into resources too, so if we later internationalize they can point at language specific versions too (once available), not terribly important though